app/views/groups/_group.html.erb

```diff
@@ -2,7 +2,7 @@
   <td><%= group.name %></td>
   <td><%= group.responsable_name %></td>
   <td><%= group.responsable_contact %></td>
-  <td><%= group.start_date %></td>
-  <td><%= group.end_date %></td>
+  <td><%= group.start_date.strftime("%d/%m/%Y") %></td>
+  <td><%= group.end_date.strftime("%d/%m/%Y") %></td>
```

Comment:
Use translations to display the format you wish to use for the dates

Reply:
Done in `cf84e3e`